docs/intro-to-coalton.md

```diff
@@ -192,9 +196,32 @@ There are convenient *syntaxes* for composing functions with the
 
 These are useful to make code less noisy.
 
-Note that since these are macros (indicated by their variadic arguments), they
-cannot be used as higher-order functions. Consider either currying or the
-`compose` function if you're thinking in that direction.
+Note that since these are macros (indicated by their variadic arguments), they cannot be used as higher-order functions. Consider either currying or the `compose` function if you're thinking in that direction.
+
+### Ignoring Function Parameters
+
+Consider the function
+
+```lisp
+(coalton-toplevel
+  (define (f x y)
+    x))
+```
+
+Here, `y` is unused and will produce a warning:
+
+```
+COMMON-LISP:WARNING: warn: Unused variable
```

Comment:
This warning looks like it was generated without the readtable enabled. Could you update this with that fixed?

Reply:
This was in the REPL FWIW. Yes I'll update it.